diagnostics-app/README.md

```diff
@@ -0,0 +1,27 @@
+# DiagnosticsApp
```

Comment:
Please describe what this file represents.

Is this the PWA high level folder? if so, please rename it so that it becomes self-descriptive.

Reply:
Updated to `Diagnostics Progressive Web App`